scripts/qa_crowbarsetup.sh

```diff
@@ -4056,6 +4056,29 @@ function onadmin_test_setuphost
     safely $SCRIPTS_DIR/mkcloud setuphost cleanup prepare setupadmin
 }
 
+function oncontroller_test_keystone_password
+{
+    safely openstack token issue
+}
+
+function update_keystone_password
+{
+    local old_password=$(crowbar keystone proposal show default | rubyjsonparse "puts j['attributes']['keystone']['admin']['password']")
+    local updated_password='barcrow'
+    crowbar batch export keystone | ruby -ryaml -e '
+y = YAML.load(ARGF)
+y["proposals"].first["attributes"]["admin"] ||= {}
+y["proposals"].first["attributes"]["admin"]["updated_password"] = "barcrow"
```

Comment:
I guess it should be used here @toabctl :smile:

Reply:
Woops, yes, fixed